src/lib/holocene/pagination.svelte

```diff
@@ -82,38 +82,46 @@
           <Icon name="chevron-right" />
         </button>
       </div>
-      <slot name="action" />
+      <slot name="action-top-right" />
```

Comment:
> Let's add a top-left action slot too

@Alex-Tideman I didn't add one there as we currently have `Updating…` text in that area. I could just put the action to the left or right of it though 🤔 Any preference?

<img width="941" alt="Screen Shot 2022-09-08 at 3 54 36 PM" src="https://user-images.githubusercontent.com/15069288/189232786-56b5a9b7-b7bf-4609-a0c3-8340fb820ec8.png">

Reply:
**Update:** I've adding it to the right of the text 👇 

| Updating | Not Updating |
| ---- | ---- |
| <img width="886" alt="Screen Shot 2022-09-08 at 4 11 19 PM" src="https://user-images.githubusercontent.com/15069288/189235008-ae4f23dd-f4fe-4eb6-acbf-f3e8244a1294.png">| <img width="853" alt="Screen Shot 2022-09-08 at 4 09 20 PM" src="https://user-images.githubusercontent.com/15069288/189234928-b6350e6c-22a1-4177-b265-e89075dffc4b.png">|

Reply:
I think what we could do is:

      {#if updating}
        <p class="mr-6 text-gray-600">Updating…</p>
      {:else}
      <slot name="action-top-left" />
      {/if} 

See how that looks and if it's too much of a flash of content.

I think eventual end state of the updating state would be to use the a <Skeleton> in each row

Reply:
Done ✅ It's not too much of a flash, but I definitely don't love hiding and then showing the button. Might think about refactoring to `<Skeleton>` in another PR 🤔

Reply:
Let's show this to Candace in our meeting, see what she thinks.

Reply:
**Update:** We can stick with this solution for now.